extensions/amp-story/1.0/amp-story.js

```diff
@@ -344,6 +346,21 @@ export class AmpStory extends AMP.BaseElement {
         createPseudoLocale(LocalizedStringsEn, (s) => `[${s} one two]`)
       );
 
+    // If there are inline strings, register as current document language.
+    const inlineStrings = this.win.document.querySelector(
+      'script[amp-strings="amp-story"]'
+    );
+    if (
+      inlineStrings &&
+      inlineStrings.getAttribute('i-amphtml-version') ==
+        getMode(this.win).rtvVersion
+    ) {
+      getLocalizationService(this.element).registerLocalizedStringBundle(
+        this.win.document.querySelector('[lang]')?.getAttribute('lang') || 'en',
```

Comment:
We need to get one language + the fallback one (English).
In the case of a Spanish document, if a new string is not translated yet, you're losing the fallback to English.

The JSON should be structured `{"en" : {}, "es": {}}` etc, so we can get both the current and fallback languages.

Reply:
Discussed offline, the inlined JSON will be a merge of all the fallbacks so that we don't need to give the story multiple languages.